src/Tribe/Admin/Activation_Page.php

```diff
@@ -47,6 +47,10 @@ public function __construct( array $args = array() ) {
 	 * Listen for opportunities to show update and welcome splash pages.
 	 */
 	public function hooks() {
+		if ( true === ( (bool) get_option( 'tribe_skip_welcome' ) ) ) {
```

Comment:
Lets save it as a `tribe_update_option`?

Reply:
Save it? You mean changing the option name?

Reply:
And pass it via `tribe_is_truthy`

Reply:
Oh, now I get it.
No, using `tribe_update_option` would mean dealing and have to update a serialized option which is non-trivial.
The idea here is to be able to run `wp option add tribe_skip_welcome 1` outside of PHP.

Reply:
`tribe_is_truthy` I can do, but the option has to be flat one to make sense; not a serialized one.

Reply:
Changed to use `tribe_is_truthy` for the check